src/Gateways/PagSeguro/CreditCard.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace BeerAndCodeTeam\PaymentGateway\Gateways\PagSeguro;
+
+class CreditCard implements PaymentMethodInterface
+{
+    private int $installments;
+    private bool $capture;
+    private string $cardNumber;
+    private string $cardExpMonth;
+    private string $cardExpYear;
+    private string $cardSecurityCode;
+    private string $holderName;
+
+    public function toArray(): array
+    {
+        return [
+            'type' => 'CREDIT_CARD',
+            'installments' => $this->installments,
+            'capture' => $this->capture,
+            'card' => [
+                'number' => $this->cardNumber,
+                'exp_month' => $this->cardExpMonth,
+                'exp_year' => $this->cardExpYear,
+                'security_code' => $this->cardSecurityCode,
+                'holder' => [
+                    'name' => $this->holderName
+                ]
+            ]
+        ];
+    }
+    public function setInstallments(int $installments)
+    {
+        if ($installments < 0) {
+            throw new \Exception("Error installment less than zero ", $installments);
```

Comment:
mensagem de erro deve ajudar o usuario: "Error: installment must be higher than zero"